workspace-manager-clienttests/README.md

```diff
@@ -49,6 +49,28 @@ Perf Test
 ```
 ./gradlew  runTest --args="suites/BasicPerf.json /tmp/TR"
 ```
+
+#### Run resiliency test
+To run an resiliency test case, specify the path of the test configuration under the `suites` or `configs` directory as shown in the following examples.
+In order to run resiliency tests, an in-cluster namespaced Service Account with special RBAC role is required.
+Please refer to document on Test Runner Service Accounts for details:
+https://docs.google.com/document/d/1-fGZqtwEUVRMmfeZfUVrn2V3M2D_eonNLepuX0ve6nM/edit?usp=sharing
+ 
+
+Resiliency Test
+```
+./gradlew  runTest --args="suites/BasicResilency.json /tmp/TR"
+./gradlew  runTest --args="configs/resiliency/DeleteInitialPods.json /tmp/TR"
+```
```

Comment:
nit: two spaces between "gradlew" and "runTest"

It would be nice to explain what the arguments are here.

Reply:
Added explanations for the two arguments in the `runTest` task.